### DISCUSSION TITLE
Issues being "converted" to discussions results in them being deleted and no way to find the new discussion

### DISCUSSION THREAD

**Discussion by willhughes-au** [concourse/concourse#5727] [General]
I've been following Issue #575 (Allow disabling `check_every` entirely for resources) for a while (since approximately December - I can no longer tell). 

I got an update a few hours ago with a new comment from @vito starting off with "Converting this into a Discussion per the new framing of Issues vs Discussions." 

That's all fine, however now I can't find that issue at all, nor the discussion.  
Clicking on the github issue link results in me getting a "This issue is deleted".  

The list of discussions only has one conversation updated in the last few hours, and that's about Postgres.  Browsing the other discussions, I can't find it, and nor can search for any obvious keywords. 

I don't know what the conversion process involves, but if possible - can you just close the original ticket and link to the discussion instead?

**Comment by jamieklassen** [comment]
looks like it moved to https://github.com/concourse/rfcs/discussions/60? my browser just automatically redirected me when I tried to visit https://github.com/concourse/concourse/discussions/575... Interestingly https://github.com/concourse/concourse/issues/575 is broken, as you say. This might be feedback for the discussions feature itself...

**Reply by willhughes-au** [reply]
Thanks.  Yeah all the links in email are for https://github.com/concourse/concourse/issues/575 - and Github isn't telling me I have a comment on a discussion in that repo.

**Comment by vito** [comment]
Huh, good to know - I'll be more careful with that feature. I converted it to a Discussion and then transferred it to the `rfcs` repo, so something in that sequence of actions must have "broken" the link. I'll experiment with this next time - maybe if I transfer first and then convert it'd be OK? 🤔
